web/satellite/src/components/accessGrants/AccessGrantsItem2.vue

```diff
@@ -33,31 +40,54 @@
 
 <script lang="ts">
 import { Component, Prop, Vue } from 'vue-property-decorator';
-
+import TrashIconWhite from '@/../static/images/accessGrants/trashIcon.svg';
+import TrashIconBlack from '@/../static/images/accessGrants/trashIcon-black.svg';
 import { AccessGrant } from '@/types/accessGrants';
 
+import { ACCESS_GRANTS_ACTIONS } from '@/store/modules/accessGrants';
+
+const {
+    TOGGLE_SELECTION,
+} = ACCESS_GRANTS_ACTIONS;
+
 // @vue/component
 @Component({
     components: {
+        TrashIconWhite,
+        TrashIconBlack,
     },
 })
 export default class AccessGrantsItem extends Vue {
     @Prop({ default: new AccessGrant('', '', new Date(), '') })
+    @Prop({default: ''})
+
     private readonly itemData: AccessGrant;
     private popupVisible = false;
+    private popupHover = false;
```

Comment:
name doesn't match boolean type

Reply:
Hi Nikolai, are you looking for me to declare this a boolean type or change the name?

I changed  both so that it now looks like this: private isPopupHovered: boolean = false;